README.md

```diff
@@ -20,25 +20,25 @@ Support...
 |--------------------------------------------|-----------------------|-----------------------|----------------------------------|--------------------|---------------------------------|------------------------------|----------------------------|---------------------------------|---------------------|---------------------|--------------------------------------------|
 | Archetype based                            | :x:                   | :x:                   | :x:                              | :heavy_check_mark: | :heavy_check_mark:              | :x:                          | :heavy_check_mark:?        | :x:?                            | :x:                 | :x:                 | Archetype based                            |
 | Filter/Query operations                    | Has/Not<br/>(Inc/Exc) | Has/Not<br/>(Inc/Exc) | Has/Not<br/>Or/Nor               |                    | Has/Not<br/>(With/<br/>Without) | Has/Not, Any<br/>(All/None)  | Has/Not, Any               | Has/Not<br/>(With/<br/>Without) | Has                 | Has                 | Filter/Query operations                    |
-| Components Type support                    | struct<br/>/class     | struct<br/>/class?    | struct<br/>/class<br/>/interface | struct?            | struct                          | class?                       | struct                     | class                           | class               | class?              | Components Type support                    |
+| Components Type support                    | struct                | struct<br/>/class?    | struct<br/>/class<br/>/interface | struct?            | struct                          | class?                       | struct                     | class                           | class               | class?              | Components Type support                    |

```

Comment:
https://github.com/Leopotam/ecs/tree/classes-based
I could make the "class" link to this^, or you don't like having class be as "supported" here.

Reply:
Its very old and obsoleted. Even `leoecs classic on structs` will not be updated anymore - only bugfixes and maintenance.